### PR TITLE
Use centroid for aperture photometry

### PR DESCRIPTION
**Pull request by juanep97**
This corrects against small astrometric inaccuracies.

### CODE REVIEW

File: iop4lib/instruments/instrument.py [hunk 1]
```diff
@@ -518,35 +518,26 @@ def compute_aperture_photometry(cls, redf, aperpix, r_in, r_out):
...
         error = calc_total_error(img, bkg.background_rms, cls.gain_e_adu)
 
         for astrosource in redf.sources_in_field.all():
             for pairs, wcs in (('O', redf.wcs1), ('E', redf.wcs2)) if redf.has_pairs else (('O',redf.wcs),):
 
-                ap = CircularAperture(astrosource.coord.to_pixel(wcs), r=aperpix)
-                annulus = CircularAnnulus(astrosource.coord.to_pixel(wcs), r_in=r_in, r_out=r_out)
+                wcs_px_pos = astrosource.coord.to_pixel(wcs)
+
+                logger.debug(f"{redf}: computing aperture photometry for {astrosource}")
```
Comment (morcuended) [inline iop4lib/instruments/instrument.py:533]: Do you want to print out the whole `redf` object?

**Review by morcuended — COMMENTED** [review]
some minor comments, otherwise looks good

File: iop4lib/instruments/instrument.py (hunk 1 above)
Comment (juanep97) [reply iop4lib/instruments/instrument.py:533]: This should print the `__str__` representation, only the id and the file location.

File: iop4lib/instruments/instrument.py [hunk 2]
```diff
@@ -518,35 +518,26 @@ def compute_aperture_photometry(cls, redf, aperpix, r_in, r_out):
...
+
+                from photutils.centroids import centroid_com, centroid_sources
```
[the remark this answers is not in the document]
Comment (juanep97) [reply iop4lib/instruments/instrument.py]: Done.

**Review by morcuended — APPROVED** [review]
looks good to me

**Pull request merged**